bundle/ScheduledVisibility/ScheduledVisibilityInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Netgen\Bundle\IbexaScheduledVisibilityBundle\Enums\HandlerType;
+
+interface ScheduledVisibilityInterface
+{
+    public function hide(Content $content): void;
+
+    public function reveal(Content $content): void;
+
+    public function isHidden(Content $content): bool;
```

Comment:
While at it let's add `isVisible()` as a negation of `isHidden()`, and as a syntax sugar. Also, in case of `Location` handler (or `ContentAndLocation` when renamed), it will be more logical to implement `isVisible()` and negate it in `isHidden()` than the other way around.

Reply:
fixed in 16f5f6f28154ef69948f2965fad53d9240e65d7d